components/form/events.md

```diff
@@ -16,13 +16,15 @@ The Form component for Blazor exposes events that allow you to respond to user a
 * [OnValidSubmit](#onvalidsubmit)
 * [OnInvalidSubmit](#oninvalidsubmit)
 
+>note The examples in this article use the `EditContext`, but you can use a model instead. 
```

Comment:
maybe link these to the sections in the overview articel?

Reply:
Great idea, done